mmv1/templates/terraform/pre_update/privateca_certificate_authority.go.tmpl

```diff
@@ -41,3 +41,19 @@ if d.HasChange("desired_state") {
 		}
 	}
 }
+
+
+// `subordinateConfig.certificateAuthority` is not directly passed 
+// to the backend when activating a sub-CA. Instead, it is used to sign CA cert
+// and activate the sub-CA at client side. See b/332548736 for details.
+// Drop this field to avoid both `subordinateConfig.certificateAuthority` 
+// and `subordinateConfig.pemIssuerChain` to be passed to the backend.
+if _, ok := obj["subordinateConfig"]; ok {
+	subConfig := obj["subordinateConfig"].(map[string]interface{})
+	// There could be case which a sub-CA was activated via `subordinateConfig.certificateAuthority`
+	// directly by older version of providers.
+	// For backward compatibility, delete `certificateAuthority` only if `pemIssuerChain` is presented.
+	if _, ok := subConfig["pemIssuerChain"]; ok {
+		delete(subConfig, "certificateAuthority")
```

Comment:
`certificateAuthority` was not be sent to API backend. It was only used to fetch `pemIssuerChain` [which is the value specified when calling backend](https://github.com/gfxcc/magic-modules/blob/fe5261738873baeb94443099ccd7fdcc1e3ad1c9/mmv1/third_party/terraform/services/privateca/privateca_ca_utils.go#L106-L107). 
With that, `certificateAuthority` only stay locally.

Reply:
API does retain this value
https://cloud.google.com/certificate-authority-service/docs/reference/rest/v1/projects.locations.caPools.certificateAuthorities#SubordinateConfig

and we do read it from the API. https://github.com/modular-magician/terraform-provider-google-beta/blob/8205ebe14bc69d903151fea9c192ddfb56c6b43a/google-beta/services/privateca/resource_privateca_certificate_authority.go#L1091

Just because the API isn't using the value doesn't mean it doesn't retain a state for it. This call will cause a diff between API state and users config.